homepage/homepage_publishing_client.go

```diff
@@ -13,34 +13,34 @@ import (
 	"github.com/ONSdigital/log.go/v2/log"
 )
 
-type HomepageClienter interface {
+type Clienter interface {
 	AddNavigationCache(ctx context.Context, updateInterval time.Duration) error
 	GetHomePage(ctx context.Context, userAccessToken, collectionID, lang string) (*model.HomepageData, error)
 	GetNavigationData(ctx context.Context, lang string) (*topicModel.Navigation, error)
 	Close()
 	StartBackgroundUpdate(ctx context.Context, errorChannel chan error)
 }
 
-type HomepagePublishingClient struct {
-	HomepageUpdater
+type PublishingClient struct {
+	Updater
 	navigationCache *cache.NavigationCache
 	languages       []string
 }
 
-func NewHomePagePublishingClient(ctx context.Context, clients *Clients, languages []string) HomepageClienter {
-	return &HomepagePublishingClient{
-		HomepageUpdater: HomepageUpdater{
+func NewHomePagePublishingClient(ctx context.Context, clients *Clients, languages []string) Clienter {
```

Comment:
If we've changed the struct to just `PublishingClient` we should probably rename this related function too.
```suggestion
func NewPublishingClient(ctx context.Context, clients *Clients, languages []string) Clienter {
```